### ISSUE TITLE
Model performance

### ISSUE THREAD

**Issue by TJL2018GH** [magnumical/GCN_for_EEG#7]
Hello, researcher
I got your mat file in the last Issues. Then I ran onEEGcode.py, but the accuracy of the validation set was only 24.8, and after 100 epochs, the accuracy and los remained unchanged. Is this a problem with your model?

**Comment by magnumical** [comment]
Hi,
The results vary by changing parameters in `onEEGcode.py `.
I set them in a way to check my code is working or not! I also achieved around 42% acc with around 200 epochs

**Comment by TJL2018GH** [comment]
Thank you very much for your answer. You mean that the highest accuracy
rate of this model will not exceed 50%, right? The performance gap between
the GCN model and the CNN model is quite large. Is GCN not suitable for
this task? In the open source code of EEG_DL, I found that the accuracy
rate in their papers can reach 90%, for example, GCNs-Net. Why is the gap
so big?

Reza Amini <notifications@github.com> 于2021年1月16日周六 上午2:59写道：

**Comment by magnumical** [comment]
I just mentioned the highest acc I achieved. I set everything in minimum to with respect to my machine! You can change parameters and obtain that 90%. 
Actually GCNs and CNNs have similar performance.

**Comment by TJL2018GH** [comment]
Thanks for your answer. I modified some parameters, such as learning rate
and batch size, but the accuracy is still below 30%. The most important
thing is that the accuracy and loss have not changed at all, neither
increased nor decreased. I think the accuracy rate is difficult to exceed
50%. What parameters have you modified?

Reza Amini <notifications@github.com> 于2021年1月16日周六 下午2:41写道：

**Comment by TJL2018GH** [comment]
Is there some mistakes in the data and label of the mat file?

一念清净,烈焰成池 <tianjilong2018@gmail.com> 于2021年1月16日周六 下午5:46写道：

**Comment by magnumical** [comment]
You should reconsider numbers related to layers! 
I mean such parameters : 
```
params['F'] = [16, 32, 64, 128, 256, 512]  # Number of graph convolutional filters.
params['K'] = [2, 2, 2, 2, 2, 2]           # Polynomial orders.
params['p'] = [2, 2, 2, 2, 2, 2]           # Pooling sizes.
params['M'] = [4]                          # Output dimensionality of fully connected layers.

```
